trtools/prancSTR/README.rst

```diff
@@ -132,4 +132,4 @@ Below are :code:`prancSTR` examples using HipSTR VCFs. Data files can be found a
 
 Citations
 ----------------
-The latest manuscript for citation of the tool can be found at https://doi.org/10.1101/2023.11.22.568371
\ No newline at end of file
+If you utilize prancSTR for your work please cite: Aarushi Sehgal, Helyaneh Ziaei Jam, Andrew Shen, Melissa Gymrek, Genome-wide detection of somatic mosaicism at short tandem repeats, Bioinformatics, Volume 40, Issue 8, August 2024, btae485, https://doi.org/10.1093/bioinformatics/btae485
```

Comment:
Just a suggestion to have this on its own line. Feel free to ignore if you prefer it the original way

```suggestion
If you utilize prancSTR for your work please cite:

> Aarushi Sehgal, Helyaneh Ziaei Jam, Andrew Shen, Melissa Gymrek, Genome-wide detection of somatic mosaicism at short tandem repeats, Bioinformatics, Volume 40, Issue 8, August 2024, btae485, https://doi.org/10.1093/bioinformatics/btae485
```